src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -117,6 +116,10 @@ def make_call(client: Any, endpoint: str, **kwargs):
         if e.response["error"] == "user_not_in_channel":
             raise TryAgain
 
+        # NOTE we've experienced a wide range of issues when Slack's performance is degraded
+        if e.response["error"] == "fatal_error":
```

Comment:
I think we should add a wait here, if the service is downgraded 5 requests in a row aren't likely to succeed.

Reply:
Just here under this `if` or should we add `wait=wait_exponential(multiplier=x, min=y, max=z)` to the decorator, so it would also apply to the other cases? The one where we would probably wait for longer than we should would be when we get rate limited.

Reply:
I'm not sure if there is a clean way to mix and match (while keeping wait for retry), so I was going to just add a simple wait time.